Xamarin.Forms.Core.iOS.UITests/BaseTestFixture.cs

```diff
@@ -50,43 +50,33 @@ protected virtual void FixtureSetup ()
 #pragma warning disable 618
 		[TestFixtureTearDown]
 #pragma warning restore 618
-		protected virtual void FixtureTeardown ()
-		{	
+		protected virtual void FixtureTeardown()
+		{
 		}
 
 		[SetUp]
-		protected virtual void TestSetup () 
+		protected virtual void TestSetup()
 		{
-			if (!ShouldResetPerFixture) {
-				RelaunchApp ();
+			if (!ShouldResetPerFixture)
+			{
+
+				RelaunchApp();
 			}
-			App.Screenshot ("Begin Test");
 		}
 
 		[TearDown]
-		protected virtual void TestTearDown ()
+		protected virtual void TestTearDown()
 		{
-			App.Screenshot ("Test complete");
+
 		}
 
-		void RelaunchApp ()
+		void RelaunchApp()
 		{
 			App = null;
-			RunningApp.App = null;
-
-			try {
-				RunningApp.Restart ();
-			} catch (Exception ex) {
-				// if at first you dont succeed
-				RunningApp.Restart ();
-			}
-
-			// Wrap the app in ScreenshotConditional so it only takes screenshots if the SCREENSHOTS symbol is specified
-			App = new ScreenshotConditionalApp(RunningApp.App);
-
-			App.SetOrientationPortrait ();
-			ScreenBounds = App.RootViewRect ();
-			NavigateToGallery ();
```

Comment:
could you avoid style "fixes", it greatly reduce the SNR and makes the PR harder to review. thanks